.github/workflows/build-images.yaml

```diff
@@ -38,17 +38,21 @@ jobs:
           else
           echo "VERSION=${{ github.sha }}" >> $GITHUB_ENV
           fi
-      - name: Install qemu dependency
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y qemu-user-static
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.IMG_REGISTRY_USERNAME }}
+          password: ${{ secrets.IMG_REGISTRY_TOKEN }}
```

Comment:
Any particular issue you hit that may have motivated adding this step?

Reply:
Didn't face any issue . I have added the login before we build/pushed the image. so that it should fail before build action get invoked

Reply:
@guicassolato Please let me know if i have to do any further changes. I am going to raise similar kind of changes for authorino operator as well

Reply:
I see the point. Authentication is hardly the step we fail here though. I'm tempted to say we don't really need it.

I see some repetition with the parameters we have to pass to `redhat-actions/push-to-registry@v2` further down as well.

Reply:
ok. I will update it

Reply:
@guicassolato Removed it & updated the code changes